sdks/python/apache_beam/pipeline_test.py

```diff
@@ -602,6 +603,12 @@ def test_timestamp_param_map(self):
           p | Create([1, 2]) | beam.Map(lambda _, t=DoFn.TimestampParam: t),
           equal_to([MIN_TIMESTAMP, MIN_TIMESTAMP]))
 
+  def test_pane_info_param(self):
```

Comment:
We can add 1 more test with actual pane.

Reply:
Done. This actually caught a bug, which I fixed. Thanks!